src/js/tabs/pid_tuning.js

```diff
@@ -399,6 +407,30 @@ TABS.pid_tuning.initialize = function (callback) {
             $('.idleMinRpm').hide();
         }
 
+        if (semver.gte(CONFIG.apiVersion, "1.43.0")) {
+            const ratesTypeListElement = $('select[id="ratesType"]'); // generates list
+            let ratesList;
+            ratesList = [
+                {name: "Betaflight"},
+                {name: "Raceflight"},
+                {name: "KISS"},
+                {name: "Actual"},
+                {name: "QuickRates"},
+            ];
+            // add future rates types here with CONFIG.apiVersion check
+            for (let i = 0; i < ratesList.length; i++) {
+                ratesTypeListElement.append('<option value="' + (i) + '">' + ratesList[i].name + '</option>');
```

Comment:
Change this string concatenation to use template strings.

Reply:
I forgot about it, sorry